Assets/Scripts/Editor/TagSelectorPropertyDrawer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using UnityEngine;
+using UnityEditor;
+using System.Collections.Generic;
+ 
+//Original by DYLAN ENGELMAN http://jupiterlighthousestudio.com/custom-inspectors-unity/
```

Comment:
Did you add these refs? In any case, since this is borrowed code, **please make a detailed account of how it's being used, why we're using it, and what changes we've made to it.**